docs/3.x.x/getting-started/quick-start.md

```diff
@@ -1,285 +1,281 @@
+
+   The **Relation** Field
+
+   Click the **"+ Add New Field"** button.
+
+   From here, click the **Relation** field.
+
+   This will bring you to the **Add New Relation** screen. We will change two elements on this page for the **Relation** field.
+
+   The first element to change is **Permission (Users-Permissions)** to **Restaurant** (on the right).
+
+   The second element to change is to click the **Many to Many** icon (in the middle). It should now read, **"Categories has and belongs to many Restaurants"**.
+
+   Then click the **Continue** button.
+
+4. Lastly, we will save this Content Type. Click the **Save** button.
+
+5. Wait for Strapi to restart.
+
+Verify in the left menu under **CONTENT TYPES**, that you see **Categories**, **Restaurants** and **Users**.
+
+We are now ready to [manage and add data to the Content Types](#_4-manage-and-add-data-to-content-type) we just created.
+
+Next, click on the **Strapi** logo (top left corner) to return the main administration panel.
+
+::: warning NOTE
+See the [CLI documentation](../cli/CLI.md#strapi-generateapi) for more information on how to add Content Types the hacker way.
 :::
 
 ### Files structure
 
-A new directory has been created in the `./api` folder of your application which contains all the needed stuff for your `Post` Content Type: routes, controllers, services and models. Take a look at the [API structure documentation](../concepts/concepts.md#files-structure) for more informations.
+A new directory has been created in the `./api` folder of your application which contains all the needed files related to your `restaurant` and `category` Content Types: routes, controllers, services and models. Take a look at the [API structure documentation](../concepts/concepts.md#files-structure) for more information.
+
+---
+
+## 4. Manage and add data to Content Type
+
+After creating [the Content Types](#_3-create-a-content-type), we now need to manage and add data/content to the new Content Types.
+
+### Add data to Content Type
+
+1. We will create an entry of a restaurant called, **"Strapi Restaurant"** with a description saying, **"Strapi restaurant is a cosy restaurant delivering one of the very fastest and nicest dining experiences in the world, combining nods to tradition with fierce modernity, warmth with daring."**.
+
+   Click on **Restaurants** under the **CONTENT TYPES** menu in order to **Add New Restaurant**.
+
+   Next click on the **+ Add New Restaurant** button (in the top right corner). Go ahead and type **"Strapi Restaurant"** in the **Name** field, and type the content (above) into the **Description** field.
 
+   Then press the **Save** button (in the top right corner).
 
-**Well done, you created your first Content Type using Strapi!**
+2. When it is properly saved, you will see your restaurant listed in the entries. From here you can edit it or add a new resturant.
```

Comment:
You will see your restaurants and your categories listed in the entries, from where you can edit, add or delete any of them.

Reply:
Yeah haha, I edited this already.